integration/airflow/openlineage/airflow/utils.py

```diff
@@ -258,11 +258,11 @@ def get_job_name(task):
 
 
 def get_custom_facets(
-    task, is_external_trigger: bool, task_instance: "TaskInstance" = None
+    dagrun, task, is_external_trigger: bool, task_instance: "TaskInstance" = None
 ) -> Dict[str, Any]:
     custom_facets = {
         "airflow_runArgs": AirflowRunArgsRunFacet(is_external_trigger),
-        "airflow_version": AirflowVersionRunFacet.from_task(task),
+        "airflow_version": AirflowVersionRunFacet.from_dagrun_and_task(dagrun, task),
```

Comment:
The `TaskInstance` has a reference to both the `DagRun` and the `Task` - you could simplify this by passing in the `task_instance` directly. Is it ever actually `None`? or is it always available?

Reply:
It is actually `None` the most cases but the call from the listener. For example, airflow 1.1 integration calls it when the DAG is registered and at that moment the `TaskInstance` is not available.

Reply:
the airflow 1.x integration is being dropped very soon, so maybe we can revisit this in a while